packages/design-system/src/components/label.tsx

```diff
@@ -122,6 +122,7 @@ type Props = {
 } & ComponentProps<typeof StyledLabel>;
 
 export const isLabelButton = (color: Props["color"]) =>
+  color === "default" ||
```

Comment:
From what I see it's always true now. Can be removed

Reply:
I think the check still fails if the color is `undefined` Which renders the label without a button.

```jsx
{isButton ? <button>{children}</button> : children}
```

<img width="829" alt="Screenshot 2023-12-21 at 00 16 56" src="https://github.com/webstudio-is/webstudio/assets/11075561/d72bd155-e134-49ea-b1b7-e78a4bdb3c31">

should it be left as it is ?. Because `Label` is still used for headings for individual properties in `transition-content` and `box-shadow-content` etc where no color is sent to them ?

Reply:
@istarkov instead cahnged teh check a little. It's being exported and used outside too. Let me know, if it's fiene now.